src/components/mapSettingsMenu/MenuActivator.vue

```diff
@@ -0,0 +1,90 @@
+<template>
+    <v-theme-provider theme="dark">
+        <v-btn
+            v-if="appConfig && mapSettings.indicator"
+            role="button"
+            :class="props.isLargeScreen ? 'indicator-menu-activator-desktop' : 'indicator-menu-activator-mobile'"
+        >
+            <p class="text-wrap">
+                {{ buttonSummary }}
+            </p>
+            <v-icon :icon="menuOpen ? 'mdi-chevron-down' : 'mdi-chevron-up'" end></v-icon>
+            <v-menu
+                :class="props.isLargeScreen ? 'indicator-menu-desktop' : 'indicator-menu-mobile'"
+                activator="parent"
+                v-model="menuOpen"
+                :close-on-content-click="false"
+                :close-on-back="false"
+            >
+                <v-list
+                    class="bg-transparent opacity-80 elevation-0"
+                    :class="isLargeScreen ? 'indicator-menu-list-item-desktop' : 'indicator-menu-list-item-mobile'"
+                >
+                    <v-list-item
+                        class="bg-black mb-2 rounded elevation-4 border-lg indicator-menu-list-item-wrapper"
+                        :ripple="false"
+                    >
+                        <slot></slot>
+                    </v-list-item>
+                </v-list>
+            </v-menu>
+        </v-btn>
+    </v-theme-provider>
+</template>
+
+<script setup lang="ts">
+import { ref } from "vue";
+import { storeToRefs } from "pinia";
+import { useAppStore } from "../../stores/appStore";
+
+const menuOpen = ref(false);
+const { appConfig, mapSettings, admin0GeojsonFeature } = storeToRefs(useAppStore());
+
+const props = defineProps({
+    isLargeScreen: {
+        type: Boolean,
+        required: true
+    }
+});
+
+const buttonSummary = computed(() => {
+    const isGlobal = !admin0GeojsonFeature.value?.properties.COUNTRY;
+    const country = admin0GeojsonFeature.value?.properties.COUNTRY || "Global";
+    const indicator = appConfig.value.indicators[mapSettings.value.indicator].humanReadableName;
+    const adminLevel = mapSettings.value.adminLevel === 1 ? "Admin 1" : "Admin 2";
+    return isGlobal ? `${country} | ${indicator}` : `${country} | ${indicator} | ${adminLevel}`;
```

Comment:
This character is not vertically centered in the font we use.

Some alternative characters to the pipe `|` for your consideration:

1. `｜` - in the font used, this results in a grey line that is better vertically centered than the pipe
2. `│` - a taller, well-vertically centered one, that doesn't come out grey

I sourced these alternatives from https://en.wikipedia.org/wiki/Vertical_bar#Unicode_code_points

Reply:
I prefer number 1

Reply:
I suppose another approach would be have a slightly different (but still dark) background for any text, with a little bit of padding between the 'sections'. That might look a little busy though, and probably (even) weird(er) with wrapping text.

Reply:
But also, just toning down the colour of the pipe to grey will also make it seem less distracting I think.

Reply:
have taken this to the slack channel!